exec/java-exec/src/main/java/org/apache/drill/exec/record/selection/SelectionVector4.java

```diff
@@ -119,15 +119,38 @@ public boolean next() {
     return true;
   }
 
+  public boolean isEmpty() {
```

Comment:
I would expect any method named isXYZ() to return a boolean without modifying the internal state.  What was wrong with leaving it the way it is?

Reply:
Reverted the change. Earlier did it to use isEmpty() method but it was not needed later on.